docs/source/help-support.rst

```diff
@@ -1,35 +1,49 @@
-=================
+================
 Help and support
-=================
+================
 
-* Get free community support with our `Google group
-  <https://groups.google.com/a/anaconda.com/forum/#!forum/anaconda>`_.
+Community support
+======================
 
-* Paid `support <https://www.anaconda.com/support/>`_,
-  `training <https://www.anaconda.com/training/>`_ and
-  `consulting <https://www.anaconda.com/consulting/>`_
-  options are available. Our teaching philosophy is that the best
-  way to learn is with hands-on experience addressing real-world
-  problems. Courses are available to individuals online, at
-  numerous sites, or in-house at your place of business. We also
-  offer consulting services for the analysis, management and
-  visualization of scientific and business data or optimizing
-  your processing workflows on modern hardware and GPUs.
+Use the following tools to ask and answer questions, discuss ways to use conda,
+request new features, and submit any other comments you might have.
 
-* Support is included in purchases of `Anaconda Enterprise
-  <https://www.anaconda.com/enterprise/>`_.
+.. grid:: 1 2 2 2
 
+  .. grid-item::
 
-Join the conda email group
-===========================
+     .. card:: Conda community forum
+        :link: https://conda.discourse.group/
+
+        Join our Discourse forum for conda discussion and news
+
+  .. grid-item::
+
+     .. card:: Conda Matrix chat
+        :link: https://matrix.to/#/#conda:matrix.org
+
+        Chat about conda issues and projects with other users
+
+  .. grid-item::
+
+     .. card:: Anaconda community forum
+        :link: https://community.anaconda.cloud/
+
+        Ask and answer questions about Anaconda's products
 
-Join the mailing lists for both `Anaconda
-<https://groups.google.com/a/anaconda.com/forum/#!forum/anaconda>`_
-and `conda
-<https://groups.google.com/a/anaconda.com/forum/#!forum/conda>`_.
-Ask questions, answer questions, discuss ways to use conda,
-request new features and submit any other comments you may have.
+Paid support
+============
 
+The conda organization does not directly offer paid support, training, or
+consultation on conda package management or creation. However, this section
+lists organizations that do provide such services. If your organization would
+like to be added to this list, please open a pull request against the conda-docs repository.
```

Comment:
I think it would be helpful to link to the repo directly here (please double-check my edit suggestion for proper formatting!)
```suggestion
like to be added to this list, please open a pull request against the `conda-docs repository<https://github.com/conda/conda-docs>`_.
```